std_msgs/Header header
std_msgs/UInt64 planeId

# global plane equation ax+by+cz+d=0
std_msgs/Float64 a
std_msgs/Float64 b
std_msgs/Float64 c
std_msgs/Float64 d